content/en/docs/armory-agent/agent-options.md

```diff
@@ -11,38 +11,58 @@ description: >
 
 ## Where to configure the Agent
 
-Set these options at the agent level in the `kubesvc.yaml` configuration file. If deploying as a non-Spinnaker<sup>TM</sup> service, you need to specify a `clouddriver.grpc` endpoint (e.g. `grpc.spinnaker.example.com:443`).
+On Kubernetes, configure the Agent using a `ConfigMap`. See the Quickstart's [Configure the Agent]({{< ref "armory-agent-quick#configure-the-agent" >}}) section for an example.
 
-## Kubernetes account
+If deploying as a non-Armory-Enterprise service, you need to specify a `clouddriver.grpc` endpoint (e.g. `grpc.spinnaker.example.com:443`) in your `kubesvc.yaml` file.
```

Comment:
The config `clouddriver.grpc` needs to be specified regardless of deploying as an Armory Enterprise service or not.

Reply:
That's in the original text that Nico wrote. Looking at the example URL and port, I wonder if he's referring to deploying the Agent outside of Kubernetes?  I think he said he could run the Agent on his laptop. 

I removed the line because the quickstart installation is targeted only at Kubernetes.